Kernel/SubstHelper.hpp

```diff
@@ -80,12 +80,9 @@ class SubstHelper
   static Literal* apply(Literal* lit, Applicator& applicator)
   {
     CALL("SubstHelper::apply(Literal*...)");
```

Comment:
We only actually use `sort` in the case when `subbedLit` is still a two-variable equality. Move it into there, save some compute and avoid the uninitialised mess.